build.gradle

```diff
@@ -45,6 +45,10 @@ def addCommonConfigurationForJavaModules(Project project) {
 }
 
 def addCommonConfigurationForAndroidModules(Project project) {
+    project.dependencies {
+        provided files("lib/rt-stub.jar")
```

Comment:
next change will add this automagically to provided deps in android rules if retrolambda is enabled